index.html

```diff
@@ -533,45 +597,119 @@ <h3>ecdsa-rdfc-2019</h3>
         </p>
 
         <section>
-          <h4>Add Proof (ecdsa-rdfc-2019)</h4>
+          <h4>Create Proof (ecdsa-rdfc-2019)</h4>
+
+        <p>
+The following algorithm specifies how to create a [=data integrity proof=] given
+an <a>unsecured data document</a>. Required inputs are an
+<a>unsecured data document</a> ([=map=] |unsecuredDocument|), and a set of
+proof options ([=map=] |options|). A [=data integrity proof=] ([=map=]),
+or an error, is produced as output.
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Let |proof| be an empty [=map=].
```

Comment:
Hmm, this seems a little off from how the other algorithms are written. It looks like `proof` should be a clone of the proof options, `options`. This doesn't say that additional other options (such as a JSON-LD document loader) can be passed in some other way, but maybe we don't need that to be prescriptive -- it's up to implementations to ensure that they can execute things like RDF canon however they've been written.

```suggestion
Let |proof| be a clone of |options|.
```